src/Provider/CentralBankOfCzechRepublicProvider.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of Swap.
+ *
+ * (c) Florian Voutzinos <florian@voutzinos.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swap\Provider;
+
+use Swap\Exception\UnsupportedCurrencyPairException;
+use Swap\Model\CurrencyPair;
+use Swap\Model\Rate;
+use Swap\Util\CurrencyCodes;
+use Swap\Util\StringUtil;
+
+/**
+ * Central Bank of Czech Republic (CNB) provider.
+ *
+ * @link http://tcmb.gov.tr
+ * @author Petr Kramar <petr.kramar@perlur.cz>
+ */
+class CentralBankOfCzechRepublicProvider extends AbstractProvider
+{
+    const URL = 'http://www.cnb.cz/cs/financni_trhy/devizovy_trh/kurzy_devizoveho_trhu/denni_kurz.txt';
+	const DATE_FORMAT = 'd.m.Y';
```

Comment:
Looks like you have some formatting issues. This code is not compatible with the PRS-2 standard http://www.php-fig.org/psr/psr-2/